lib/util.js

```diff
@@ -261,8 +261,7 @@ const util = {
 #endif  // WIDEVINE_CDM_VERSION_H_`
 
     // If version file or fake lib file aren't existed, create them.
-    if (!fs.existsSync(widevineConfig.widevineCdmHeaderFilePath) ||
-        !fs.existsSync(widevineConfig.widevineCdmHeaderFilePath)) {
+    if (!fs.existsSync(widevineConfig.widevineCdmHeaderFilePath) {
```

Comment:
Oops! As I wrote in above comments, another one should be `widevineConfig.fakeWidevineCdmLibFilePath`.
Although possibility of one of both are not existed is very low if it isn't deleted explicitly.
If header is existed and lib file isn't existed, build will be failed.

@PatOnTheBack Can you change another one from `widevineCdmHeaderFilePath` to `fakeWidevineCdmLibFilePath` instead of deleting?
Thanks for checking this!!

Reply:
@simonhong I fixed it.